package.json

```diff
@@ -31,7 +31,7 @@
   },
   "peerDependencies": {
     "webpack": "^2 || ^3 || ^4",
-    "html-webpack-plugin": "^2 || ^3"
+    "html-webpack-plugin": "^2 || ^3 || ^4.0.0-alpah.2"
```

Comment:
Small sp mistake here: `alpha`

We'll probably wait for the major version to drop before merging this just incase the API changes, so let's also update this simply to `^4`

Reply:
😅 Will Update.